infra/dockerfile.base

```diff
@@ -1,13 +1,12 @@
-FROM python:3.7.4-slim
+FROM alpine:latest
```

Comment:
There is a python alpine image which has Python already installed, I think it's preferable to use that one.

Reply:
Changed in the latest commit.